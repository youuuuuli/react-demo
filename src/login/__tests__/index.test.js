import React from 'react';
import { render, screen } from '@testing-library/react';
import Login from '../index';

it('renders Login Page', () => {
  const { container } = render(<Login />);
  const linkElement = screen.getByText(/Log-in to your account/i);

  expect(linkElement).toBeInTheDocument();
  expect(container.firstChild).toMatchSnapshot();
});
