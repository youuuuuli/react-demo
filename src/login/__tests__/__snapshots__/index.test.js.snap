// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login Page 1`] = `
<div
  class="ui center aligned middle aligned grid"
  style="height: 100vh;"
>
  <div
    class="column"
    style="max-width: 450px;"
  >
    <h2
      class="ui teal center aligned header"
    >
      <i
        aria-hidden="true"
        class="user circle icon"
      />
      Log-in to your account
    </h2>
    <form
      class="ui large form"
    >
      <div
        class="ui stacked segment"
      >
        <div
          class="field"
        >
          <div
            class="ui fluid left icon input"
          >
            <input
              placeholder="Username"
              type="text"
              value=""
            />
            <i
              aria-hidden="true"
              class="user icon"
            />
          </div>
        </div>
        <div
          class="field"
        >
          <div
            class="ui fluid left icon input"
          >
            <input
              placeholder="Password"
              type="password"
              value=""
            />
            <i
              aria-hidden="true"
              class="lock icon"
            />
          </div>
        </div>
        <button
          class="ui teal large fluid button"
        >
          Login
        </button>
      </div>
    </form>
  </div>
</div>
`;
